LinkifiedText/index.jsx

```diff
@@ -49,6 +49,7 @@ LinkifiedText.propTypes = {
       indices: PropTypes.arrayOf(React.PropTypes.number),
     }),
   ),
+  size: PropTypes.string,
```

Comment:
So I don't think the `PropTypes.string` is quite enough here. Maybe we could grab it from `Text.propTypes.size`. That would show that there is a link for that prop between the Components so someone could trace it back.